docs/source/guide/get_started/quick_start_guide/cli_commands.rst

```diff
@@ -353,6 +353,50 @@ The command below will evaluate the trained model on the provided dataset:
     it is possible to pass both PyTorch weights ``.pth`` or OpenVINO™ IR ``openvino.xml`` to ``--load-weights`` option.
 
 
+***********
+Explanation
+***********
+
+``otx explain`` runs the explanation of a model on the specific dataset.
```

Comment:
```suggestion
``otx explain`` runs the explanation algorithm of a model on the specific dataset. It helps explain model's decision-making process in a way that is easily understood by humans.
```

Reply:
I try to give more intuitions what is the "explain"

Reply:
thanks Kirill.